src/containers/lineup/CardLineup.module.css

```diff
@@ -18,6 +18,8 @@
   margin-bottom: 5px;
 }
 
-.emptyMobileContainer {
-  width: 174px;
+.emptyMobileCard {
+  width: 100%;
+  min-width: 174px;
+  max-width: 190px;
```

Comment:
how did you get the 190 number?

Reply:
the above three come from the styles of the card: https://github.com/AudiusProject/audius-client/blob/46948cea5a468d2bfdb58d189139c1b6a672f29c/src/components/card/mobile/Card.module.css#L18 is there a good way for us to reference these styles? maybe make an empty card component and put it in the Card.tsx file?

Reply:
Ah yeah -- might be nice to put the empty component there then and then also share via a css variable like https://github.dev/AudiusProject/audius-client/blob/637c9b4f337c117ee54ecead959693e77990e825/src/containers/App.module.css#L19